utils/bcm27xx-eeprom/Makefile

```diff
@@ -1,18 +1,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bcm27xx-eeprom
-PKG_VERSION:=v2024.04.20-2712
+PKG_VERSION:=2024.06.05
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/raspberrypi/rpi-eeprom/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=83ea92e64d9a620376ef081d69f3cdde5a8b376b4a56aeb685f8a56dd10e7b14
+PKG_SOURCE_URL:=https://codeload.github.com/raspberrypi/rpi-eeprom/tar.gz/v$(PKG_VERSION)-2712?
```

Comment:
Better to use local tarballs IMO.

Reply:
What do you mean by "local tarballs?"

Reply:
PKG_SOURCE_PROTO:=git
PKG_SOURCE_DATE:=2024-06-05

Reply:
Done in 8834e2f12abeaa70f8d763b6a77610136fe5d340